src/shared/parse-helpers.h

```diff
@@ -3,12 +3,28 @@
 
 #include <stdint.h>
 
+#include "list.h"
+
 enum {
         PATH_CHECK_FATAL    = 1 << 0,  /* If not set, then error message is appended with 'ignoring'. */
         PATH_CHECK_ABSOLUTE = 1 << 1,
         PATH_CHECK_RELATIVE = 1 << 2,
 };
 
+typedef enum OpenFileFlags {
+        OPENFILE_RDONLY         = 1 << 0,
+        _OPENFILE_MAX,
+        _OPENFILE_INVALID  = -EINVAL,
+        _OPENFILE_MASK_PUBLIC   = OPENFILE_RDONLY,
```

Comment:
Please define this after `OPENFILE_RDONLY`, similar to how `PortableFlags` does it